.github/workflows/zcashd-cd.yml

```
@@ -0,0 +1,77 @@
+name: Zcashd Manual Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        default: 'testnet'
+      size:
+        default: 10
+
+env:
+  PROJECT_ID: zealous-zebra
+  REGION: us-east1
+  ZONE: us-east1-b
```

us-east1 sometimes gets full with our stuff / approaches quota with our existing zebrad deployments, so I'd suggest a different region:

```suggestion
  REGION: us-central1
  ZONE: us-central1-c
```